src/styles/content.scss

```diff
@@ -461,3 +461,45 @@ $arrow-svg: url("data:image/svg+xml,%3Csvg width='16' height='9' viewBox='0 0 16
   font-size: .875rem;
   margin-bottom: 0;
 }
+
+// User Message
+.os-raise-usermessage-link {
+  display: flex;
+  align-items: center;
+  border: .125rem solid $os-dark-green;
+  border-radius: .5rem;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 24 24'%3E%3Cpath fill='%230a5b50' d='M10.59 13.41c.41.39.41 1.03 0 1.42c-.39.39-1.03.39-1.42 0a5.003 5.003 0 0 1 0-7.07l3.54-3.54a5.003 5.003 0 0 1 7.07 0a5.003 5.003 0 0 1 0 7.07l-1.49 1.49c.01-.82-.12-1.64-.4-2.42l.47-.48a2.98 2.98 0 0 0 0-4.24a2.98 2.98 0 0 0-4.24 0l-3.53 3.53a2.98 2.98 0 0 0 0 4.24m2.82-4.24c.39-.39 1.03-.39 1.42 0a5.003 5.003 0 0 1 0 7.07l-3.54 3.54a5.003 5.003 0 0 1-7.07 0a5.003 5.003 0 0 1 0-7.07l1.49-1.49c-.01.82.12 1.64.4 2.43l-.47.47a2.98 2.98 0 0 0 0 4.24a2.98 2.98 0 0 0 4.24 0l3.53-3.53a2.98 2.98 0 0 0 0-4.24a.973.973 0 0 1 0-1.42'/%3E%3C/svg%3E");
+  background-repeat: no-repeat;
+  background-position: 1rem 50%;
+  background-size: 1.5rem;
+  min-height: 3.5rem;
+  padding-left: 3.5rem;
+  margin-bottom: 1rem;
+
+}
+
+.os-raise-usermessage-lightbulb {
+  display: flex;
+  align-items: center;
+  border: .125rem solid $os-dark-green;
+  border-radius: .5rem;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='1em' height='1em' viewBox='0 0 24 24'%3E%3Cpath fill='%230a5b50' d='M20 11h3v2h-3zM1 11h3v2H1zM13 1v3h-2V1zM4.92 3.5l2.13 2.14l-1.42 1.41L3.5 4.93zm12.03 2.13l2.12-2.13l1.43 1.43l-2.13 2.12zM12 6a6 6 0 0 1 6 6c0 2.22-1.21 4.16-3 5.2V19a1 1 0 0 1-1 1h-4a1 1 0 0 1-1-1v-1.8c-1.79-1.04-3-2.98-3-5.2a6 6 0 0 1 6-6m2 15v1a1 1 0 0 1-1 1h-2a1 1 0 0 1-1-1v-1zm-3-3h2v-2.13c1.73-.44 3-2.01 3-3.87a4 4 0 0 0-4-4a4 4 0 0 0-4 4c0 1.86 1.27 3.43 3 3.87z'/%3E%3C/svg%3E");
+  background-repeat: no-repeat;
+  background-position: 1rem 50%;
+  background-size: 1.5rem;
+  min-height: 3.5rem;
+  padding-left: 3.5rem;
+  margin-bottom: 1rem;
+
+}
+
+.os-raise-usermessage-link p,  .os-raise-usermessage-lightbulb p {
+  color: $raise-light-black;
+  font-weight: 700;
+  font-size: .875rem;
+  margin-bottom: 0;
+}
+
+.os-raise-usermessage-link a, .os-raise-usermessage-lightbulb a {
+  text-decoration: underline;
+}
```

Comment:
We're introducing a bit of asymmetry here as it relates to styling links in these boxes:

![image](https://github.com/openstax/k12-apps-raise/assets/839025/2545601c-1758-4c7d-b65f-991915cdc79f)


Can we confirm with UX whether :point_up: is intentional, or if we should be adding `.os-raise-familysupport` here like:

```css
.os-raise-usermessage-link a,
.os-raise-usermessage-lightbulb a,
.os-raise-familysupport a {
  text-decoration: underline;
}
```

Reply:
Yes they should be underlined. Thank you @rnathuji good catch!